### PR TITLE
Extract apple_framework_relative_headers from https://github.com/material-foundation/material-internationalization-ios.

### PR DESCRIPTION
**Pull request by jverkoey**
This source was forked from https://github.com/material-foundation/material-internationalization-ios/blob/94a37db3477e6de6b8c3b40e524478056b73031a/apple_framework_relative_headers.bzl

Moving these rules to a shared repo allows other repos to make use of this functionality in a consistent manner. This work is part of rolling out https://github.com/material-components/material-components-ios/issues/5491

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019-present The Material Foundation Authors. All Rights Reserved.
```
Comment (romoore) [inline BUILD]: Date?

Comment (jverkoey) [reply BUILD]: Done.

**Review by romoore — COMMENTED** [review]
If this code was copied from another repository in the organization with a different copyright header, do we need to do anything to denote that?

**Comment by jverkoey** [comment]
I believe the copyright headers are the same; are you seeing a difference somewhere?

**Review by yarneo — APPROVED** [review]

**Comment by romoore** [comment]
The copyright difference is that the original copyright was from 2019 while these are 2020. My question was whether we have to do anything to reference that this was taken from another project and not created within this project as an original work?

**Comment by jverkoey** [comment]
> My question was whether we have to do anything to reference that this was taken from another project and not created within this project as an original work?

I've added a note to the readme to clarify the source of the code.

**Pull request merged**